### PR TITLE
bugfix:modify dfget output

### LINKED ISSUE #817: [bug]The dfget cli output is ugly
## Why you need it?

```
dfget version:workspace:/root/.small-dragonfly sign:15458-1565830210.799client:127.0.0.1 connected to node:127.0.3.0:8004start download by dragonflydownload SUCCESS(0) cost:2.868s length:21976647 reason:0
--2019-08-15 08:50:14--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:workspace:/root/.small-dragonfly sign:15571-1565830214.708client:127.0.0.1 connected to node:127.0.2.0:8003start download by dragonflydownload SUCCESS(0) cost:2.552s length:21976647 reason:0
--2019-08-15 08:50:18--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:workspace:/root/.small-dragonfly sign:15678-1565830218.353client:127.0.0.1 connected to node:127.0.3.0:8004start download by dragonflydownload SUCCESS(0) cost:3.679s length:21976647 reason:0
--2019-08-15 08:50:23--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:workspace:/root/.small-dragonfly sign:15795-1565830223.225client:127.0.0.1 connected to node:127.0.3.0:8004start download by dragonflydownload SUCCESS(0) cost:2.156s length:21976647 reason:0
--2019-08-15 08:50:26--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:workspace:/root/.small-dragonfly sign:15912-1565830226.292client:127.0.0.1 connected to node:127.0.3.0:8004start download by dragonflydownload SUCCESS(0) cost:1.645s length:21976647 reason:0
--2019-08-15 08:50:29--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:workspace:/root/.small-dragonfly sign:16025-1565830229.005client:127.0.0.1 connected to node:127.0.3.0:8004start download by dragonflydownload SUCCESS(0) cost:3.674s length:21976647 reason:0
```

The dfget cli is ugly,no pause ,i can't find the information i want clearly,

### PR DESCRIPTION
**Pull request by yunfeiyangbuaa**
Signed-off-by: yunfeiyangbuaa <yunfeiyang@buaa.edu.cn>

### Ⅰ. Describe what this PR did
make dfget output clear

### Ⅱ. Does this pull request fix one issue?

fixes #817 

### Ⅴ. Special notes for reviews

now the out  put is
```
--2019-08-22 19:28:46--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:
workspace:10156-1566473326.503
sign:10156-1566473326.503
client:127.0.0.1 connected to node:127.0.1.0:8002
start download by dragonfly...
download SUCCESS !
cost:3.333s length:21976647

--2019-08-22 19:28:08--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:
workspace:7650-1566473288.932
sign:7650-1566473288.932
download FAIL(1104)
cost:0.142s length:-1
reason:1001
error:{"Code":1104,"Msg":"download fail and not back source: 1001"}
exit status 80


```

### CODE REVIEW

**Comment by yunfeiyangbuaa** [comment]
> use `pkg/printer` instead of `fmt`

I have changed fmt to pkg/printer. @lowzj

File: cmd/dfget/app/root.go [hunk 1]
```diff
@@ -299,12 +299,12 @@ func handleNodes() error {
...
 func resultMsg(cfg *config.Config, end time.Time, e *errortypes.DfError) string {
 	if e != nil {
-		return fmt.Sprintf("download FAIL(%d) cost:%.3fs length:%d reason:%d error:%v",
+		return fmt.Sprintf("download FAIL(%d)\ncost:%.3fs length:%d\nreason:%d\nerror:%v",
 			e.Code, end.Sub(cfg.StartTime).Seconds(), cfg.RV.FileLength,
 			cfg.BackSourceReason, e)
 	}
-	return fmt.Sprintf("download SUCCESS(0) cost:%.3fs length:%d reason:%d",
-		end.Sub(cfg.StartTime).Seconds(), cfg.RV.FileLength, cfg.BackSourceReason)
+	return fmt.Sprintf("download SUCCESS !\ncost:%.3fs length:%d",
```
Comment (lowzj) [inline cmd/dfget/app/root.go]: output them in one line and don't change the content, `0` is the return code, it represents success.

Comment (yunfeiyangbuaa) [reply cmd/dfget/app/root.go]: I think success(0) will make others confused, success or failed?

Comment (yunfeiyangbuaa) [reply cmd/dfget/app/root.go]: success don't need a code to present it is seccess, success is 0, 0 is success.but success(0) make people confused.At least when I first use Dragonfly,I was confused with it

Comment (yunfeiyangbuaa) [reply cmd/dfget/app/root.go]: I think it is better to be ```
fmt.Sprintf("download SUCCESS cost:%.3fs length:%d reason:%d",
		end.Sub(cfg.StartTime).Seconds(), cfg.RV.FileLength, cfg.BackSourceReason)
```

Comment (yunfeiyangbuaa) [reply cmd/dfget/app/root.go]:
```
--2019-08-22 20:27:58--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:
workspace:4889-1566476878.241
sign:4889-1566476878.241
download FAIL(1104) cost:0.373s length:-1 reason:1001 error:{"Code":1104,"Msg":"download fail and not back source: 1001"}
exit status 80

--2019-08-22 20:28:40--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:
workspace:5126-1566476920.745
sign:5126-1566476920.745
client:127.0.0.1 connected to node:127.0.1.0:8001
start download by dragonfly...
download SUCCESS cost:4.380s length:21976647 reason:0

```

File: dfget/core/core.go [hunk 2]
```diff
@@ -79,8 +79,9 @@ func Start(cfg *config.Config) *errortypes.DfError {
 
 // prepare the RV-related information and create the corresponding files.
 func prepare(cfg *config.Config) (err error) {
-	fmt.Printf("dfget version:%s", version.DFGetVersion)
-	fmt.Printf("workspace:%s sign:%s", cfg.WorkHome, cfg.Sign)
+	printer.Printf("dfget version:%s", version.DFGetVersion)
+	printer.Printf("workspace:%s", cfg.Sign)
```
Comment (lowzj) [inline dfget/core/core.go]: `cfg.Sign` -> `cfg.WorkHome`

**Comment by yunfeiyangbuaa** [comment]
>     * change `fmt` to `pkg/printer` in `BackDownloader`
> 
>     * the example in `dfget/app/root` should be updated
Done  @lowzj

**Comment by lowzj** [comment]
LGTM

**Pull request merged**